ecommerce/api.py

```diff
@@ -223,6 +224,7 @@ def generate_cybersource_sa_payload(order, redirect_url):
         "access_key": settings.CYBERSOURCE_ACCESS_KEY,
         "amount": str(order.total_price_paid),
         "currency": "USD",
+        "customer_ip_address": ip_address,
```

Comment:
Can we only include this field if the `ip_address` variable is not empty? Cybersource has silently errored before if certain fields are included but are empty.